.github/workflows/dashboards_cypress_workflow.yml

```diff
@@ -0,0 +1,143 @@
+  SECURITY_ENABLED: ${{ inputs.SECURITY_ENABLED != '' && inputs.SECURITY_ENABLED || 'false' }}
+  SPEC: 'cypress/integration/core_opensearch_dashboards/*'
+
+jobs:
+  cypress-tests:
+    runs-on: ubuntu-latest
+    container:
+      image: docker://opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v2
+      options: --user 1001
+    env:
+      # prevents extra Cypress installation progress messages
+      CI: 1
+      # avoid warnings like "tput: No value for $TERM and no -T specified"
+      TERM: xterm
+    name: Run cypress tests ${{ inputs.UNIQUE_ID}}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          path: ./${{ env.OSD_PATH }}
+          repository: ${{ env.TEST_REPO }}
+          ref: '${{ env.TEST_BRANCH }}'
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: './${{ env.OSD_PATH }}/.nvmrc'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Setup Yarn
+        run: |
+          npm uninstall -g yarn
+          npm i -g yarn@1.22.10
+          yarn config set network-timeout 1000000 -g
+
+      - name: Get Cypress version
+        id: cypress_version
+        run: |
+          echo "name=cypress_version::$(cat ./${{ env.OSD_PATH }}/package.json | jq '.devDependencies.cypress' | tr -d '"')" >> $GITHUB_OUTPUT
+    
+      - name: Cache Cypress
+        id: cache-cypress
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/Cypress
+          key: cypress-cache-v2-${{ runner.os }}-${{ hashFiles('**/package.json') }}
+        env:
+          CYPRESS_INSTALL_BINARY: ${{ steps.cypress_version.outputs.cypress_version }}
+      - run: npx cypress cache list
+      - run: npx cypress cache path
+
+      - name: Get package version (Linux)
+        run: |
+          cd ${{ env.OSD_PATH }}
+          echo "VERSION=$(yarn --silent pkg-version)" >> $GITHUB_ENV
+
+      - name: Run bootstrap
+        run: |
+          cd ${{ env.OSD_PATH }}
+          yarn osd bootstrap
+
+      - name: Download and extract Opensearch artifacts
+        run: |
+          CWD=$(pwd)
+          mkdir -p $CWD/${{ env.OPENSEARCH_DIR }}
+          source ${{ env.OSD_PATH }}/scripts/common/utils.sh
+          open_artifact $CWD/${{ env.OPENSEARCH_DIR }} ${{ env.OPENSEARCH }}
+      
+      - name: Download and extract Opensearch Dashboards artifacts
+        run: |
+          CWD=$(pwd)
+          mkdir -p $CWD/${{ env.DASHBOARDS_DIR }}
+          source ${{ env.OSD_PATH }}/scripts/common/utils.sh
+          open_artifact $CWD/${{ env.DASHBOARDS_DIR }} ${{ env.DASHBOARDS }}
+
+      - name: Run Cypress tests 
+        run: |
+          source ${{ env.OSD_PATH }}/scripts/cypress_tests.sh
+          run_dashboards_cypress_tests  
+
+      # Screenshots are only captured on failures
+      - uses: actions/upload-artifact@v3
+        if: failure()
+        with:
+          name: osd-cypress-screenshots
```

Comment:
same as the above comment: indicate that these tests were executed against a release artifact so `release-osd-cypress-screenshots` or something if folks were looking at a glance.